modules/042-kube-dns/openapi/config-values.yaml

```diff
@@ -58,6 +68,15 @@ properties:
           maximum: 3600
           description: |
             Max TTL in seconds for NOERROR responses.
+        transportProtocolMode:
+          type: string
+          description: |
+            The mode defines the transport protocol for communicating with the upstream DNS server:
+            - `PreferUDP` - Uses UDP, even if the original request came over TCP. However, if the DNS response exceeded 512 bytes, it will be truncated and marked with the TC bit, as specified in [RFC 1035](https://datatracker.ietf.org/doc/html/rfc1035.html#section-4.2.1). Then based on [RFC 5966](https://datatracker.ietf.org/doc/html/rfc5966), the request will be resent over TCP. If there are any other UDP problems, the request will not be resent over TCP.
```

Comment:
```suggestion
            - `PreferUDP` - Uses UDP, even if the original request came over TCP. However, if the DNS response exceeded 512 bytes, it will be truncated and marked with the TC bit by upstream, as specified in [RFC 1035](https://datatracker.ietf.org/doc/html/rfc1035.html#section-4.2.1). Then based on [RFC 5966](https://datatracker.ietf.org/doc/html/rfc5966), the request will be resent over TCP. If there are any other UDP issues, the request will not be resent over TCP.
```